tests/integration/mixer/policy/ratelimiting_test.go

```diff
@@ -330,7 +330,7 @@ func testsetup(ctx resource.Context) error {
 		return err
 	}
 	redInst = &red
-	ing, err := ingress.New(ctx, ingress.Config{Istio: ist})
+	ing, err := ingress.New(ctx, ingress.Config{Istio: ist, IngressType: ingress.PlainText})
```

Comment:
no need to change if the plaintext equals to 0 or "". that's go's default field value.

Reply:
Good point. Thanks!